src/main/groovy/be/cytomine/software/consumer/Main.groovy

```diff
@@ -87,7 +87,7 @@ class Main {
     }
 
     static void ping() {
-        int limit = 20
+        int limit = 100
```

Comment:
Why do you updated these values ? Was it for testing purpose ? If yes, it shouldn't be committed.